.travis.yml

```diff
@@ -123,6 +117,7 @@ install:
       cabal new-update head.hackage -v
     fi
   - grep -Ev -- '^\s*--' ${HOME}/.cabal/config | grep -Ev '^\s*$'
+  - if [ $HCNUMVER -eq 80403 ]; then cabal new-install -w ${HC} --symlink-bindir=$HOME/.local/bin hlint --constraint='hlint ==2.1.*'; fi
```

Comment:
One thing to keep in mind here is that building `hlint` from scratch will contribute to the build time. Are we sure we don't want to just install it from `apt-get`?

Reply:
it's cached, dependencies are somewhat stable, and it's only single job (no reason to `hlint` in every matrix sub-job)

Reply:
... and you get newer hlint (or actually you can control which version you get, with `--hlint-version`)

Reply:
Alright, you've convinced me :)

One more question about this: it looks like the script currently builds all of the dependencies for `hlint` sequentially. Can this be made parallel?